packages/lab/src/pill-next/PillNext.css

```diff
@@ -0,0 +1,89 @@
+/* Styles applied to the root element */
+.saltPillNext {
+  --pill-background: var(--saltPillNext-background, var(--salt-actionable-primary-background));
```

Comment:
replace all with --pillNext

Reply:
I don’t think this is necessary since it’s a local variable. Doesn’t clash with anything.

Reply:
you would hope so but a few of us (fernanda and I) got clashes. when the values change it will start going wonky. best to be safe

Reply:
Ok, will change it. Can you give me an example of the issue? I'm struggling to come up with one myself

Reply:
idk why it happened, but on storybook with formfield and formfieldnext we were getting clashes when both used --formField- 
even though in form field next, the .saltFormField class wasn't used, storybook loaded them all in to the page, so they were defined globally (or wherever they were, high enough to get captured) and causing errors
@Fercas123 had the same for list

Reply:
I very much agree @joshwooding

Reply:
I removed all of the `--saltPillNext-*` variables